sdk/src/main/java/com/mparticle/messaging/PushAnalyticsReceiverCallback.java

```diff
@@ -0,0 +1,22 @@
+package com.mparticle.messaging;
+
+public interface PushAnalyticsReceiverCallback {
```

Comment:
This might make more sense to share the `MPMessagingRouter` prefix instead of the Receiver implementation.